shell/browser/electron_browser_client.cc

```diff
@@ -524,6 +524,11 @@ void ElectronBrowserClient::OverrideWebkitPrefs(
   prefs->picture_in_picture_enabled = false;
 #endif
 
+  ui::NativeTheme* native_theme = ui::NativeTheme::GetInstanceForNativeUi();
```

Comment:
should be `GetInstanceForWeb` since we use the same for deciding `ShouldUseDarkColors` in native theme api.

Reply:
That was a mistake, updated the `ShouldUseDarkColors` back to be native UI